src/v/cluster/controller_backend.cc

```diff
@@ -1342,11 +1342,6 @@ ss::future<std::error_code> controller_backend::force_abort_replica_set_update(
         co_return errc::success;
     }
 
-    // we do not have to request update/cancellation twice
-    if (current_cfg.revision_id() == rev) {
```

Comment:
Is this intentional?

Reply:
no, i was looking at an incident and accidentally removed it, good catch